docs/02-app/02-api-reference/04-functions/redirect.mdx

```diff
@@ -3,11 +3,32 @@ title: redirect
 description: API Reference for the redirect function.
 ---
 
-The `redirect` function allows you to redirect the user to another URL. If you need to redirect to a 404, you can use the [`notFound` function](/docs/app/api-reference/functions/not-found).
+The `redirect` function allows you to redirect the user to another URL. `redirect` can be used in Server Components, Client Components, [Route Handlers](/docs/app/building-your-application/routing/router-handlers), and [Server Actions](/docs/app/building-your-application/data-fetching/server-actions).
 
-## `redirect()`
+If you need to redirect to a 404, use the [`notFound` function](/docs/app/api-reference/functions/not-found) instead.
 
-Invoking the `redirect()` function throws a `NEXT_REDIRECT` error and terminates rendering of the route segment in which it was thrown. `redirect()` can be called with a relative or an absolute URL.
+## Parameters
+
+The `redirect` function accepts two arguments:
+
+```
+redirect(path, type)
+```
+
+| Parameter | Type                    | Default     | Description                                                 |
+| --------- | ----------------------- | ----------- | ----------------------------------------------------------- |
+| `path`    | `string`                | -           | The URL to redirect to. Can be a relative or absolute path. |
+| `type`    | `'replace'` or `'push'` | `'replace'` | The type of redirect to perform.                            |
+
+By default, `redirect` will replace the current URL in the browser history stack. If you want to add a new entry to the history stack (e.g. when using it in a server action to allow the user to return to the previous page), use `push` instead.
```

Comment:
Might be the same thing, but what about calling redirect in server components?

Reply:
@alvarlagerlof update the paragraph, hopefully that's clearer! 🤞🏼